shared_model/schema/queries.proto

```diff
@@ -46,6 +46,15 @@ message TxPaginationMeta {
   oneof opt_last_tx_height {
     uint64 last_tx_height = 7;
   }
+  oneof opt_tx_text{
+    uint64 tx_text = 8;
+  }
+  oneof opt_tx_account{
+    uint64 tx_account = 9;
+  }
+  oneof opt_tx_sent_from_account{
+    uint64 tx_sent_from_account = 10;
+  }
```

Comment:
In one PR I suggest to have only one feature. Those params are being added in another PR: https://github.com/hyperledger/iroha/pull/3840
so please don't add them here.